defos/src/defos_linux.cpp

```diff
@@ -512,6 +515,22 @@ void defos_update()
 
     window_has_focus = focused_window == win;
     apply_cursor_visible();
+
+	// Cursor enter/exit events
+	bool state = defos_is_mouse_in_view();
+	if (state != is_cursor_in_view && window_has_focus)
```

Comment:
The logic here is a bit messy. You should instead consider the mouse-in-view state to be `defos_is_mouse_in_view() && window_has_focus`.

Reply:
I will get to this later today.

Reply:
Tested, works fine.